src/Tribe/Importer/File_Importer.php

```diff
@@ -378,4 +403,34 @@ protected function get_featured_image( $event_id, array $record ) {
 
 		}
 	}
+
+	/**
+	 * Returns an iterator to iterate over the last created terms.
+	 *
+	 * @since TBD
+	 *
+	 * By default a NoRewindIterator will be returned, this will allow successive calls from iterating code,
+	 * e.g. a `foreach`, to resume from the previously last position.
+	 *
+	 * @param string $taxonomy
+	 * @param bool $rewind Whether to return a rewinding iterator (`true`) or a NoRewind one (`false`);
+	 *                     defaults to `false`.
+	 *
+	 * @return ArrayIterator|NoRewindIterator An ArrayIterator built on the term IDs created for the taxonomy
+	 *                                        or a NoRewindIterator built on top of it.
+	 */
+	public function created_terms( $taxonomy, $rewind = false ) {
+		$iterator = Tribe__Utils__Array::get( $this->created_terms, $taxonomy, new ArrayIterator() );
+
+		return $rewind ? $iterator : new NoRewindIterator( $iterator );
+	}
+
+	/**
+	 * Hooks on the term creation to watch for any newly created terms.
+	 *
+	 * @since TBD
+	 */
+	public function watch_term_creation() {
+		add_action( 'created_term', array( $this, 'on_created_term' ), 10, 3 );
```

Comment:
Should we check if `has_action` before adding it (potentially again)?

Reply:
yep, good idea, thanks